### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by jeanbuhendwa**
### In this pull request, I have : 

- [ ] Addd the missing JSX elements to the project without adding extra functionality
- [ ] Used the CSS rules detailed in Zeplin to style the website to match the design 
- [ ] Fix all linter errors

### CODE REVIEW

File: src/components/home.js [hunk 1]
```diff
@@ -33,18 +33,39 @@ const HomePage = () => {
...
+                    <button
+                      type="button"
+                      onClick={() => handelRemove(book.item_id)}
+                    >
+                      Remove
+                    </button>
+                    <button type="button">Edit</button>
+                  </div>
+                </div>
+                <div className="bookProgress">
```
Comment (codecaiine) [inline src/components/home.js:56]: - [ ] Good job, your design looks great 👏 But there are a few things you need, please add the missing default category on the book page is missing as you can see on the screenshot below ⬇️  Also category link uses a different color. Please fix them in order to make your project design great 👍 

Bookstore CMS (1)

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @jeanbuhendwa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good design ✔️ 
- Follow CSS best practice ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/home.js (hunk 1 above)
Comment (jeanbuhendwa) [reply src/components/home.js:56]: #### Thanks @codecaiine for your feedback

**Review by DeliceLydia — APPROVED** [review]
## Hi @jeanbuhendwa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Addressed all requested changes


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jeanbuhendwa** [comment]
#### Thanks @DeliceLydia for the approval. 😎

**Pull request merged**